src/configs/openstack.rs

```diff
@@ -0,0 +1,21 @@
+use crate::config::{ModuleConfig, RootModuleConfig};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct OspConfig<'a> {
+    pub format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for OspConfig<'a> {
+    fn new() -> Self {
+        OspConfig {
+            format: "on [$symbol$cloud(\\($project\\))]($style) ",
+            symbol: "☁️  ",
+            style: "bold yellow",
```

Comment:
Nitpick but I always thought of OpenStack as having a red/white colour scheme would it be better to use one of those colours?

Reply:
I didn't want to put a red symbol here because, to me, red means bad. But I am open to suggestions. What would be the best place to choose symbols from?

Reply:
Ah that makes sense. Normally we use emoji for the default symbols and save the more exotic icons like [nerdfont](https://www.nerdfonts.com/cheat-sheet) for the examples.

Reply:
I could not find a suitable Emoji for this. The closest one seems to be 🟥, but that's more like a "record" button than anything. I'm open to suggestions.

Reply:
Hmm, I can't see anything else either. Let's leave this as it is for now, I'm sure someone in the community will suggest something better when they start using it.